### ISSUE TITLE
Ylabel colides with tick texts.

### ISSUE THREAD

**Issue by TobiasWegener** [bendichter/brokenaxes#35]
First of all fantastic package, thank you for making it.
Small issue, the y label seem to be fixed thus when the tick texts take more room, this may lead to collisions e.g.:
```
import matplotlib.pyplot as plt
from brokenaxes import brokenaxes
import numpy as np

fig = plt.figure(figsize=(5,4))
bax = brokenaxes(xlims=((0.,.1), (.4, .7)), ylims=((-1.25, -.71), (-.79, .25)), hspace=.05)
x = np.linspace(0, 1, 100)
bax.plot(x, np.sin(10 * x), label='sin')
bax.plot(x, np.cos(10 * x), label='cos')
bax.legend(loc=3)
bax.set_xlabel('time')
bax.set_ylabel('value')
```
[Ylabel_collision.pdf](https://github.com/bendichter/brokenaxes/files/3285709/Ylabel_collision.pdf)

**Comment by bendichter** [comment]
Yes, the auto positioning was often resulting in a collision so I set a better default and gave the option of changing the offset. See `bax.set_xlabel`

**Comment by mdepitta** [comment]
Hello,
I am also noticing the same problem with the x-axis label. I cannot adjust the location meaning:

    bax.set_xlabel('x label',y=...)

does not seem to change the `y` position.
Fantastic package btw.
